<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnSistema1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Snap ITC</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sistema 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSistema2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Snap ITC</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sistema 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>390</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Snap ITC</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVolver">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>778</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Snap ITC</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Volver al Men√∫</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="vistaSimulacion">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>900</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEditar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>778</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Snap ITC</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Editar Cuerpo</string>
    </property>
   </widget>
   <zorder>vistaSimulacion</zorder>
   <zorder>btnSistema1</zorder>
   <zorder>btnSistema2</zorder>
   <zorder>btnSalir</zorder>
   <zorder>btnVolver</zorder>
   <zorder>btnEditar</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
